ts/utils/appVersion.ts

```diff
@@ -64,17 +65,18 @@ export const getAppVersion = () =>
 /**
  * return true if the app must be updated
  * @param serverInfo the backend info
+ * @param section
  */
 export const isUpdateNeeded = (
-  serverInfo: ServerInfo | undefined,
+  serverInfo: IOVersionInfo | undefined,
   section: "min_app_version_pagopa" | "min_app_version"
 ) =>
   fromNullable(serverInfo)
     .map(si => {
       const minAppVersion = Platform.select({
         default: "undefined",
-        ios: si[section].ios,
-        android: si[section].android
+        ios: si[section]?.ios ?? "undefined",
+        android: si[section]?.android ?? "undefined"
```

Comment:
```typescript
const minAppVersion = Platform.select({
        ios: si[section]?.ios,
        android: si[section]?.android
      });
return !isVersionAppSupported(minAppVersion ?? "", getAppVersion());
```
Could it be a valuable refactoring?

Reply:
Changed in 4aac6fa51d4e0cee72c7ada540b41697ba81e6a1